src/main/java/UtilsTest/PasswordsTest.java

```diff
@@ -0,0 +1,66 @@
+package UtilsTest;
+
+import Utils.Passwords;
+import Utils.Tuple;
+import jdk.nashorn.internal.ir.annotations.Ignore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class PasswordsTest {
+
+    String password;
+
+    @BeforeEach
+    void setUp() {
+        password = "Password123";
+    }
+
+    @AfterEach
+    void tearDown() {
+    }
+
+    @Test
+    void Given_Password_Then_ReturnTuple() throws InvalidKeySpecException, NoSuchAlgorithmException {
+        // Given
+
+        // When
+        Tuple<?, ?> actual = Passwords.createSaltAndHash(password);
+
+        // Then
+        assertNotNull(actual);
+        assertTrue(actual instanceof Tuple);
```

Comment:
In Junit 5 (Our testing framework) we test things using `Assertions`, e.g. `Assertions.assertTrue(actual)`. In this case `Assertions` is imported at the top of the class, so we only need to write `assertTrue(actual)`.

There are a huge number of assertions we can make, to test a wide range of things. https://junit.org/junit5/docs/5.0.1/api/org/junit/jupiter/api/Assertions.html